README.md

```diff
@@ -25,39 +25,43 @@ steps:
+By default, the branch or tag ref that triggered the workflow will be checked out. If you wish to check out a different branch, a different repository or use different token to checkout, specify that using `with.ref`, `with.repository` and `with.token`.
 
-Checkout different branch from the workflow repository:
+## Checkout different branch from the workflow repository
 ```yaml
 - uses: actions/checkout@v1
   with:
     ref: some-branch
 ```
 
-Checkout different private repository:
+## Checkout different private repository
 ```yaml
 - uses: actions/checkout@v1
   with:
     repository: myAccount/myRepository
-    ref: refs/heads/release
+    ref: refs/heads/master
     token: ${{ secrets.GitHub_PAT }} // `GitHub_PAT` is a secret contains your PAT.
 ```
+> - `${{ github.token }}` is scoped to the current repository, so if you want to checkout another repository that is private you will need to provide your own [PAT](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
 
-Checkout private submodules:
+## Checkout private submodules
 ```yaml
 - uses: actions/checkout@v1
   with:
-    submodules: recursive
+    submodules: true // 'recursive' 'true' or 'false'
     token: ${{ secrets.GitHub_PAT }} // `GitHub_PAT` is a secret contains your PAT.
 ```
-> - `with.token` will be used as `Basic` authentication header for https requests talk to https://github.com from `git(.exe)`, ensure those private submodules are configured via `https` not `ssh`.
-> - `${{ github.token }}` only has permission to the workflow triggering repository. If the repository contains any submodules that comes from private repository, you will have to add your PAT as secret and use the secret in `with.token` to make `checkout` action work.
+> - Private submodules must be configured via `https` not `ssh`.
+> - `${{ github.token }}` only has permission to the workflow triggering repository. If the repository contains any submodules that come from private repositories, you will need to add your [PAT](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) as secret and use the secret in `with.token` to make the `checkout` action work.
 
 For more details, see [Contexts and expression syntax for GitHub Actions](https://help.github.com/en/articles/contexts-and-expression-syntax-for-github-actions) and [Creating and using secrets (encrypted variables)](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables)
 
 # Changelog
 
-## v1.1.0 (unreleased)
+## v1.2.0
+- Reverted persisting auth token changes to fix a bug with custom auth flows
+
+## v1.1.0
```

Comment:
Update this with a note to do not use this behavior.  The (breaking) functionality intended with this release will be in the 2.0 release.